.gitignore

```diff
@@ -2,4 +2,5 @@ vendor/
 storage/cache/aop/*
 storage/cache/api/*
 !storage/cache/aop/.gitkeep
-.env
\ No newline at end of file
+.env
+.idea/
```

Comment:
@sandermangel feel free to remove it. I added it in there since not everyone has a global gitignore set up (like me) and PHPstorm is kind of popular.